src/components/header/header.js

```diff
@@ -23,15 +24,40 @@ export default class Header {
       }
       header = createElementsFromString(usersHeader(headersProps));
     }
-    const screenWithHeader = target.appendChild(header);
+    if (target.firstChild) {
+      replaceFirstChild(target, header);
+    } else {
+      target.appendChild(header);
+    }
     if (type === roles.manager) {
-      target.querySelector('.header-content__switch-mode-button').addEventListener('click', () => {
+      header.querySelector('.header-content__switch-mode-button').addEventListener('click', () => {
         switchMode(props);
       });
     }
-    target.querySelector('.exit-ico').addEventListener('click', () => { logout(props.router); });
+    header.querySelector('.exit-ico').addEventListener('click', () => {
+      logout(props.router);
+    });
+
+    // to remove
```

Comment:
Here is comment "to remove"
When should it be done?

Reply:
I think, when we will have real opportunity to test balance update